perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -57,16 +63,37 @@
 
 flags.DEFINE_enum(
     'scenario', 'OneByteRW', ['OneByteRW', 'ListConsistency',
-                              'SingleStreamThroughput', 'CleanupBucket'],
+                              'SingleStreamThroughput', 'CleanupBucket',
+                              'MultiStreamWrite', 'MultiStreamRead'],
     'The various scenarios to run. OneByteRW: read and write of single byte. '
     'ListConsistency: List-after-write and list-after-update consistency. '
     'SingleStreamThroughput: Throughput of single stream large object RW. '
-    'CleanupBucket: Cleans up everything in a given bucket.')
+    'CleanupBucket: Cleans up everything in a given bucket.'
+    'MultiStreamWrite: Write objects with many streams at once.'
+    'MultiStreamRead: Read objects with many streams at once.')
 
 flags.DEFINE_integer('iterations', 1, 'The number of iterations to run for the '
                      'particular test scenario. Currently only applicable to '
                      'the ListConsistency scenario, ignored in others.')
 
+flags.DEFINE_integer('objects_per_stream', 1000, 'The number of objects to '
+                     'read and write per stream in the MultiStreamThroughput '
+                     'scenario.')
+flags.DEFINE_string('object_sizes', 1, 'The size of the objects to use, as a '
```

Comment:
Done. The examples correspond to the two examples in `object_storage_service_benchmark.py`. I think they should show that we only accept bytes and what the backend distribution format is.